boot/seal.go

```diff
@@ -409,13 +409,13 @@ func bootChains(modeenv *Modeenv, method device.SealingMethod) (BootChains, erro
 	return bc, nil
 }
 
-func resealKeyToModeenvForMethod(unlocker Unlocker, method device.SealingMethod, rootdir string, modeenv *Modeenv, expectReseal bool) error {
+func resealKeyToModeenvForMethod(unlocker Unlocker, method device.SealingMethod, rootdir string, modeenv *Modeenv, expectReseal bool, revokeOldKeys bool) error {
```

Comment:
maybe we should introduce a struct for the bool as in your other PR?

Reply:
I have rebased, but not fixed that yet. It was not as easy as I thought since it is not clear where the bool belong. But I will fix it.